WordPress/src/main/java/org/wordpress/android/util/image/getters/GlideRemoteResourceViewTarget.kt

```diff
@@ -0,0 +1,116 @@
+package org.wordpress.android.util.image.getters
+
+import android.annotation.SuppressLint
+import android.graphics.Canvas
+import android.graphics.ColorFilter
+import android.graphics.PixelFormat
+import android.graphics.Rect
+import android.graphics.drawable.Animatable
+import android.graphics.drawable.Drawable
+import android.widget.TextView
+
+import com.bumptech.glide.request.Request
+import com.bumptech.glide.request.target.SizeReadyCallback
+import com.bumptech.glide.request.target.Target
+import com.bumptech.glide.request.target.ViewTarget
+import com.bumptech.glide.request.transition.Transition
+
+import org.wordpress.android.util.R
+
+internal class GlideRemoteResourceViewTarget(
```

Comment:
Similar to my comments for `GlideImageGetter`, can we add comments about what this class is about and maybe come up with a different name that doesn't start with `Glide`?

Also, is there a reason it's an internal class?

Reply:
Because it's in the main module, I don't think it'll make any difference, so I didn't think there is a reason to break consistency here, but I don't feel strongly about it, I was just curious.

> I'm not a fan of having multiple classes declaration in a single file

For me, that depends on the classes. If that class is only used by one other class and shouldn't be used otherwise and it's short enough, I actually prefer to have it in the same file.

Reply:
I agree, it definitely depends and there are many valid cases.